README.md

```diff
@@ -75,6 +75,18 @@ npm run build:fast
 
 See also: [Contributing](#Contributing)
 
+### Other First-Time Build Requirements
+
+- [node-gyp](https://github.com/nodejs/node-gyp) Requirements
+  - Ensure that the requirements listed [here](https://github.com/nodejs/node-gyp#installation) are satisfied for your environment.
```

Comment:
In addition to this, I would link to the routerlicious readme where these deps are called out. That helps clarify that these are not dependencies of the core framework and come from server dependencies. I would also clarify that these dependencies are not needed if you are following the "standard flow" (`npm i && npm run build:fast`).

I think it would be helpful to refine how fluid-build symlinks. It would be better if it let you specify which 'monorepos' you want to symlink, and operate on those grouping individually and independently.

But if we get on yarn or pnpm workspaces then maybe we can solve it in those tools.

Reply:
I went ahead and combined my notes with the ones already in the routerlicious README, and updated the root docs to be expressed in terms of workflows that include that package. Let me know what you think.